### PR TITLE
Added Redefinition of parameters with duplicate name

### PR DESCRIPTION
**Pull request by GuillaumeDievart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ compatibility breakages that are outlined below.
 * [Removal of date.timezone Warning](#removal-of-datetimezone-warning)
 * [Removal of Alternative PHP Tags](#removal-of-alternative-php-tags)
 * [Removal of Multiple Default Blocks in Switch Statements](#removal-of-multiple-default-blocks-in-switch-statements)
+* [Removal of Redefinition of parameters with duplicate name](#removal-of-redefinition-of-parameters-with-duplicate-name)
```
Comment (tpunt) [inline README.md]: Just a few minor points :)

Capitalise the title and pluralise "name", to give: "Removal of Redefinition of Parameters with Duplicate Names"

File: README.md [hunk 2]
```diff
@@ -1243,6 +1244,29 @@ This (useless) ability has now been removed and causes a fatal error.
 
 RFC: [Make defining multiple default cases in a switch a syntax error](https://wiki.php.net/rfc/switch.default.multiple)
 
+### Removal of Redefinition of parameters with duplicate name
```
Comment (tpunt) [inline README.md]: See above point

File: README.md [hunk 3]
```diff
@@ -1243,6 +1244,29 @@ This (useless) ability has now been removed and causes a fatal error.
...
+}
+
+echo foo(5, 7);
+  
+// Pre PHP 7 result
+7
+
+// PHP 7+ result
+Fatal error: Redefinition of parameter $version in /redefinition-of-parameters.php
+```
```
Comment (tpunt) [inline README.md:1265]: Please move the example above the BC Breaks section.

**Comment by tpunt** [comment]
Just a few minor things to fix before I merge it :)

**Comment by GuillaumeDievart** [comment]
Done

**Comment by tpunt** [comment]
Thanks! Merged.

**Pull request merged**